wolfssl/wolfcrypt/types.h

```diff
@@ -1466,6 +1466,19 @@ typedef struct w64wrapper {
     #endif
 
 
+    /* opaque math variable type */
+    #if defined(USE_FAST_MATH)
```

Comment:
We shouldn't need MATH_INT_T.
mp_int should be the common type name.

Reply:
My preference is to leave this opaque type code in for now.

There are several places we have includes dependencies that cause issues here. Like sp_int.h including hash.h and  wolfmath.h including random.h.

I reviewed both STM32 and QuickAssist and patching those is not easy. Also this is still used in ESP32, SE050 and PSOC6.

stm32.c

```
wolfssl/wolfcrypt/port/st/stm32.h:152:30: error: unknown type name 'mp_int'
  152 | int stm32_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
      |                              ^~~~~~
In file included from wolfssl/wolfcrypt/sha256.h:92,
                 from wolfssl/wolfcrypt/random.h:111,
                 from wolfssl/wolfcrypt/tfm.h:47,
                 from wolfssl/wolfcrypt/wolfmath.h:44,
                 from wolfssl/wolfcrypt/asn.h:46,
                 from wolfcrypt/src/hash.c:31:
```

quickassist.c:

```
In file included from ./wolfssl/wolfcrypt/hash.h:32,
                 from ./wolfssl/wolfcrypt/sp_int.h:34,
                 from ./wolfssl/wolfcrypt/wolfmath.h:48,
                 from ./wolfssl/wolfcrypt/port/intel/quickassist.h:27,
                 from ./wolfssl/wolfcrypt/port/intel/quickassist_mem.h:27,
                 from ./wolfssl/wolfcrypt/types.h:398,
                 from ./wolfssl/wolfcrypt/aes.h:37
```